src/mailchimp/mailchimp_manager.py

```diff
@@ -25,12 +30,13 @@ class MailchimpManager:
         TODO: detail public attributes
     """
 
-    def __init__(self):
+    def __init__(self, api_key, server):
```

Comment:
The parameters `api_key` and `server_key` should not be here. This causes an API change which will affect everything else using the Mailchimp manager. Just set the attributes to `None` in the constructor.